### DISCUSSION TITLE
How to affect URL for e-mail confirmation links after sign-up

### DISCUSSION THREAD

**Discussion by user72356** [supabase/supabase#12037] [Questions] [answered]
My SITE_URL is pointing to the main URL for my site, and that is the URL that Supabase uses when sending e-mails to users asking them to confirm their e-mail addresses by cliking a link. The problem is when I deploy to dev or some other environment, I don't see a way to affect which URL gets used. I wish there was a option to auto-detect the URL the sign-up request was made from, or at least a way to supply it when calling signUp(). Did I miss something?

**Comment by GaryAustin1** [comment, marked as answer]
Most of the signup/signin calls have a redirectTo, or emailRedirectTo option to set the FULL url the email should use.  You also have to set the FULL url in the URL Configuration section of Auth.  Those will then be used instead of SiteUrl.

**Reply by user72356** [reply]
Ohh yes I had missed it in the docs because of the grey on grey tabs for the function variations. I had seen just the default tab and assumed there was no redirect.
